src/EPPlus/DigitalSignatures/SignatureInfoV1.cs

```diff
@@ -70,37 +71,37 @@ internal SignatureInfoV1(XmlElement SignatureInfo1Node)
         }
 
         //Required children
-        public string SetUpId = "";
-        public string SignatureText = "";
+        internal string SetUpId = "";
```

Comment:
These should be in the format _setupId and so on, if they are internal members

Reply:
good point!